example/src/main/resources/application.properties

```diff
@@ -1 +1 @@
-openapi.service.output = build/apiDoc/api-docs.json
\ No newline at end of file
+openapi.service.output = api-docs.json
```

Comment:
please revert this change, doesn't make any sense to assign it to default value

Reply:
done